bin/check-ec2-cpu_balance.rb

```diff
@@ -90,13 +104,14 @@ def run
         next unless instance.instance_type.start_with? 't2.'
         id = instance.instance_id
         result = data id
+        tag = config[:tag] ? " (#{instance_tag(instance, config[:tag])})" : ''
         unless result.nil?
           if result < config[:critical]
             level = 2
-            messages << "#{id} is below critical threshold [#{result} < #{config[:critical]}]\n"
+            messages << "#{id}#{tag} is below critical threshold [#{result} < #{config[:critical]}]\n"
           elsif config[:warning] && result < config[:warning]
             level = 1 if level == 0
-            messages << "#{id} is below warning threshold [#{result} < #{config[:warning]}]\n"
+            messages << "#{id}#{tag} is below warning threshold [#{result} < #{config[:warning]}]\n"
```

Comment:
so it's not smushed together should we either put a space or a separator of some kind (like  `messages << "#{id}:#{tag} is below warning threshold [#{result} < #{config[:warning]}]\n"`

Reply:
Thought about that as well, but decided agains putting it in that line as the `--tag` is optional. I handle the spacing in line 107:
```tag = config[:tag] ? " (#{instance_tag(instance, config[:tag])})" : ''```
Otherwise you'd have double spaces or a separator for no real good reason in the message if you're not using the tag feature.

Reply:
makes sense.